### ISSUE TITLE
You have to hit back button twice to go back 

### ISSUE THREAD

**Issue by jumasheff** [wataruoguchi/clever-infinite-scroll#6]
First of all, I want to thank you for your work -- it saved me a lot of time. But there is one issue with the project -- when this library is active in a page, and you visit a link and want to go back, you have to click browser's back button twice.
Any ideas on why this happens?

**Comment by trevorllarson** [comment]
I would like to second this issue, this is happening site-wide for us.

**Comment by wataruoguchi** [comment]
@jumasheff @trevorllarson ok, I'll look into it. Btw PRs are always welcome 😄

**Comment by trevorllarson** [comment]
@wataruoguchi I was running through things pretty rushed today, so I didn't get to thoroughly test, but I did find a way to at least temporarily solve the issue. I commented out the setTitleAndHistory that fires on Initialize. As far as I could test on the fly, I did not notice any negative effects, but I also assume that it was set up that way for a reason. 

Hopefully that gives some direction!
